templates/scripts.tx

```diff
@@ -18,3 +18,5 @@
 	<script src="/static/js/highlight.pack.js" type="text/javascript"></script>
 	<script>hljs.initHighlightingOnLoad();</script>
 : }
+
+<script src="//jag.duckduckgo.com:5000/livereload.js"></script>
```

Comment:
Yep, we [stash a variable, `is_dev` in the root controller](https://github.com/duckduckgo/community-platform/blob/3e5a200/lib/DDGC/Web/Controller/Root.pm#L77) so you should be able to:

```
: if $is_dev {
    <script src="..."></script>
: }
```

Should the source for this script just be '/livereload.js'?